TestsCommon/RunSettings.cs

```diff
@@ -103,4 +117,19 @@ private static string InitializeParameter(TestParameters parameters, string para
             return null;
         }
     }
+    protected static bool isPlaceholderValueOrEmpty(string propertyName){
```

Comment:
nit: PascalCase and private
```suggestion
    private static bool IsPlaceholderValueOrEmpty(string propertyName){
```

Reply:
I preferred protected over private to enable inheritance of the methods in child classes.